provider/lpweb/hapi/routes.go

```diff
@@ -25,10 +25,14 @@ func Routes(r *mux.Router, deps *deps.Deps) error {
 		t:  t,
 	}
 
+	go a.watchRpc()
```

Comment:
The RPC is really handy to have in the background, otherwise on every request we'd need to find a working RPC which may add noticeable delay; Also later this can generate data for a Prometheus endpoint, which I imagine most serious SPs will want to use.